eslintrc.json

```diff
@@ -592,8 +612,14 @@
         "test/**/*.js",
         "**/*.cy.js"
       ],
+      "globals": {
+        "apos": "writable"
+      },
```

Comment:
we write on apos in tests.

Reply:
We write to `apos` backend in tests and it's not a global (it's a top level defined variable). Plugins can also write to the global (window) `apos`.